app/containers/message/Urls.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
-import { View, Text, StyleSheet } from 'react-native';
+import {
+	View,
+	Text,
+	StyleSheet
+} from 'react-native';
```

Comment:
Remove these changes.

Reply:
I included Clipboard, Vibrations, Platform modules of react-native which are not required by Url.js as they aren't required there. I included them for fixing the issue but you redirected me to our own Toast.js component, therefore, I removed the modules.
I'll revert them.